modules/otel-collector-components.adoc

```diff
@@ -199,6 +199,109 @@ The OpenCensus receiver provides backwards compatibility with the OpenCensus pro
 Wildcards with `+*+` are accepted under the `cors_allowed_origins`.
 To match any origin, enter only `+*+`.
 
+[id="k8scluster-receiver_{context}"]
+=== Kubernetes cluster receiver
+
+The Kubernetes cluster receiver is responsible for gathering cluster-level metrics and entity events from the Kubernetes API server. It utilizes the K8s API to stay informed about updates. Authentication for this receiver is exclusively supported through service accounts.
+
+.OpenTelemetry Collector custom resource with the enabled Kubernetes cluster receiver
+[source,yaml]
+----
+  receivers:
+    k8s_cluster:
+      collection_interval: 10s
+  exporters:
+    debug:
+  service:
+    pipelines:
+      metrics:
+        receivers: [k8s_cluster]
+        exporters: [debug]
+      logs/entity_events:
+        receivers: [k8s_cluster]
+        exporters: [debug]
+----
+It also needs a Service Account
+
+.Kubernetes cluster receiver service account
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: otelcontribcol
+  name: otelcontribcol
+----
+
+.Kubernetes cluster receiver service account RBAC
+[source,yaml]
+----
+piVersion: rbac.authorization.k8s.io/v1
```

Comment:
Missed the a in apiVersion. We should also add the required ClusterRoleBinding to bind the created role to the SA.

Reply:
Done